### ISSUE TITLE
HTML autocompletion tags

### ISSUE THREAD

**Issue by babobski** [Komodo/KomodoEdit#3280]
### Short Summary
The html auto completion of tags is a bit different then it was in Komodo X.
In Komodo X when I would select a html tag from the auto completion list, it would insert the first tag like this:

```
<div|
```
In Komodo 11 you will end up with:
```
<div>|</div>
```

I prefer the old behaviour over the new.
In Komodo X, after the insertion of the tag you can type `>` and this will insert the closing tag ( if you set this preference ).
Or you can add a ` `( space ) so it will show the auto completion for the attributes.

In Komodo 11 after you insertion of the tag, you can only "open" the tag, if you want to add a attribute to you're html tag. You have a extra action to preform.

### Steps to Reproduce
 * select a html tag from auto completion list

### Platform Information

Komodo IDE, version 11.0.1, build 90784, platform win32-x86

**Comment by babobski** [comment]
Just wanted to add the related forum [link](https://community.komodoide.com/t/ko-dialogs-open-how-to-show-title-text-and-prompt/3943/9) here.

**Comment by mitchell-as** [comment]
Hi, perhaps a preference would be a good idea here.

**Comment by Naatan** [comment]
I agree the old behaviour is favourable, you can get the auto tag closing just by typing the closing bracket for that start tag. I don't think a pref is necessary, let's revert to the old behaviour.
